code/modules/client/preference/character.dm

```diff
@@ -2095,14 +2095,14 @@
 	from_db = FALSE
 	return TRUE
 
-/datum/character_save/proc/init_custom_emote_text(overrides)
-	custom_emote_text = list()
-	custom_emote_text = overrides
+/datum/character_save/proc/init_custom_emotes(overrides)
+	custom_emotes = overrides
 	for(var/datum/keybinding/custom/custom_emote in GLOB.keybindings)
 		var/emote_text = overrides && overrides[custom_emote.name]
 		if(!emote_text)
-			continue
-		custom_emote_text[custom_emote.name] = emote_text
+			custom_emote.emote_text = initial(custom_emote.emote_text)
+			continue //we set anything without an override back to default, in case it isn't that
+		custom_emotes[custom_emote.name] = emote_text
 		custom_emote.emote_text = emote_text
```

Comment:
You're iterating the global custom emote here, meaning this is going to be overridden for every player.

Are you sure this is what you want to do?

Reply:
Unsure, further testing required.

Reply:
fixed